src/app/api/crud/alerts.py

```diff
@@ -30,14 +30,19 @@ async def resolve_previous_alert(device_id: int) -> Optional[AlertOut]:
     return AlertOut(**entries[0]) if len(entries) > 0 else None
 
 
-async def create_event_if_inexistant(payload: AlertIn) -> int:
+async def create_event_if_inexistant(payload: AlertIn) -> Tuple[Optional[int], bool]:
+    """Return the id of the event to be associated with the alert and a boolean flag to tell if a new event was created
+
+    Args:
+        payload: alert object
+
+    Returns: tuple (bool, int) -> (new alert ?, event_id)
```

Comment:
it returns tuple(int, bool) instead of tuple (bool, int) right ?

Reply:
Indeed! Fixed